app/webpacker/styles/registration.scss

```diff
@@ -0,0 +1,3 @@
+.confirmed-banner {
+  margin: 15px;
```

Comment:
Nice :)

Reply:
We can have a chat when I implement updates to the design but yes, I think it's better to keep these separate rather than having a form with conditionals and all that.